selfdrive/car/tests/test_models.py

```diff
@@ -451,6 +466,11 @@ def test_panda_safety_carstate(self):
     failed_checks = {k: v for k, v in checks.items() if v > 0}
     self.assertFalse(len(failed_checks), f"panda safety doesn't agree with openpilot: {failed_checks}")
 
+  @pytest.mark.skipif(not CI, reason="When running in CI we want to make sure all the routes are uploaded to the preserved CI bucket.")
+  def test_route_on_ci_bucket(self):
+    assert self.test_route_on_bucket, "Route not on CI bucket. \
+                                       This is fine to fail for WIP car ports, just let us know and we can upload your routes to the CI bucket."
+
```

Comment:
@jnewb1 this fails on test_car_model.py, also the error message is oddly formatted:

```
======================================================================
FAIL: test_route_on_ci_bucket (__main__.CarModelTestCase.test_route_on_ci_bucket)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/batman/openpilot/openpilot/selfdrive/car/tests/test_models.py", line 472, in test_route_on_ci_bucket
    assert self.test_route_on_bucket, "Route not on CI bucket. \
AssertionError: Route not on CI bucket.                                        This is fine to fail for WIP car ports, just let us know and we can upload your routes to the CI bucket.

----------------------------------------------------------------------
Ran 8 tests in 10.034s
```

Reply:
hmm, I think it's just because its not skipped with pytest, perhaps we want to switch all unittest.main to pytest.main?